src/main/webapp/js/email-browser.js

```diff
@@ -357,9 +528,22 @@ var jasig = jasig || {};
             dataFunction: getEmailFunction(that, that.options.folderName),
             dataLengthFunction: function() { return account.accountSummary ? account.accountSummary.totalMessageCount : 0; }
         };
-
         that.pager = unicon.batchedpager(that.locate("emailList"), batchOptions);
 
+        // Notify the server of changes to pageSize so they can be remembered and update the container min-height class
+        that.pager.pager.events.initiatePageSizeChange.addListener(function(newPageSize) {
+            // console.log("newPageSize: "+ newPageSize);
+            /* if #right-content-email has .emailpreview-container-min-height-XX class remove it */
```

Comment:
would using `.emailpreview-container.min-height-xx` rather than `.emailpreview-container-min-height-xx` simplify this logic any? or would it be about the same?